OptionSelector/CQCategory.h

```diff
@@ -0,0 +1,17 @@
+//
+//  CQCategory.h
+//  Glycemic Index of Common Foods OptionSelector App
+//
+//  Created by Ayuna Vogel on 8/10/15.
+//  Copyright (c) 2015 Ayuna Vogel. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+@interface CQCategory : NSObject
```

Comment:
That's very cool! Thank you Caleb so much!! This is very very helpful. I will add comments to my classes from now on.